tendrl/commons/__init__.py

```diff
@@ -85,7 +85,7 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
         if self.current_ns.objects:
             LOG.info("Validating registered (.py) objects in %s.objects", raw_ns)
             defined_objs = defs.get("objects", {})
-            regd_objs = [obj_name for obj_name in self.current_ns.objects if not self._get_object(obj_name).internal]
+            regd_objs = [obj_name for obj_name in self.current_ns.objects if not hasattr(self._get_object(obj_name), "internal") and not obj_name.startswith('_')]
```

Comment:
ok, these are private namespaces which contain the flows and atoms of that object

instead of fetching objects using "self.current_ns.objects", please implement a method in https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L217

def _get_objects(self):
return [obj_name for obj_name in self.current_ns.objects if not obj_name.startswith('_')]